test/integration/api/audits.js

```diff
@@ -382,6 +382,23 @@ describe('/audits', () => {
           data: { age: '77', first_name: 'Alan' }
         })
         .expect(200);
+
+      await asAlice.post('/v1/projects/1/forms?publish=true')
+        .send(testData.forms.updateEntity)
+        .set('Content-Type', 'application/xml')
+        .expect(200);
+
+      // all properties changed
+      await asAlice.post('/v1/projects/1/forms/updateEntity/submissions')
+        .send(testData.instances.updateEntity.one)
+        .set('Content-Type', 'application/xml')
+        .expect(200);
+
+      await exhaust(container);
+
+      await asAlice.patch('/v1/projects/1/datasets/people/entities/12345678-1234-4123-8234-123456789abc?resolve=true')
```

Comment:
Is this suggesting that you can resolve a conflict even if there is no conflict? It seems like it shouldn't log anything if you try to resolve a conflict that doesn't exist.